tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala

```diff
@@ -112,7 +112,7 @@ class HoverTermSuite extends BaseHoverSuite {
   )
 
   check(
-    "interpolator-apply",
+    "interpolator-apply".tag(IgnoreScala3),
```

Comment:
This seems to have worked before, why was it ignored?

Reply:
Oh, I guess I overlooked it. It should be fixed!